addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -270,7 +271,6 @@
     </template>
 
     <template id="portal_my_task_planned_hours_template">
-        <strong t-if="task.planned_hours > 0">Allocated Time:</strong>
```

Comment:
If any customer customization is based on this it will break. The stable version does not allow us to do this.Adding invisible attributes or conditions, like 1=0, is a possibility. We will remove it from the master during the forward branch.